tsconfig.json

```diff
@@ -23,7 +23,9 @@
       "@mui/x-date-pickers-pro/*": ["./packages/x-date-pickers-pro/src/*"],
       "@mui/x-license-pro": ["./packages/x-license-pro/src"],
       "@mui/x-license-pro/*": ["./packages/x-license-pro/src/*"],
-      "test/*": ["./test/*"]
+      "test/*": ["./test/*"],
+      "docs/*": ["./node_modules/@mui/monorepo/docs"],
+      "docsx/*": ["./docs/*"]
```

Comment:
Removes a warning on my IDE, it was not finding the components like `PickersGrid` because the `docsx` alias did not exist in TypeScript, only in `babel`